packages/teams-js/src/public/appInitialization.ts

```diff
@@ -1,59 +1,59 @@
 import { app } from './app';
 
 /**
- * @deprecated with TeamsJS v2 upgrades
+ * @deprecated As of 2.0.0-beta.1. Please use {@link app} namespace instead.
  */
 export namespace appInitialization {
   /**
-   * @deprecated with TeamsJS v2 upgrades
+   * @deprecated As of 2.0.0-beta.1. Please use {@link app.Messages} instead.
    */
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   export import Messages = app.Messages;
   /**
-   * @deprecated with TeamsJS v2 upgrades
+   * @deprecated As of 2.0.0-beta.1. Please use {@link app.FailedReason} instead.
    */
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   export import FailedReason = app.FailedReason;
   /**
-   * @deprecated with TeamsJS v2 upgrades
+   * @deprecated As of 2.0.0-beta.1. Please use {@link app.ExpectedFailureReason} instead.
    */
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   export import ExpectedFailureReason = app.ExpectedFailureReason;
   /**
-   * @deprecated with TeamsJS v2 upgrades
+   * @deprecated As of 2.0.0-beta.1. Please use {@link app.IFailedRequest} instead.
    */
   export import IFailedRequest = app.IFailedRequest;
   /**
-   * @deprecated with TeamsJS v2 upgrades
+   * @deprecated As of 2.0.0-beta.1. Please use {@link app.IExpectedFailureRequest} instead.
    */
   export import IExpectedFailureRequest = app.IExpectedFailureRequest;
 
   /**
-   * @deprecated with TeamsJS v2 upgrades
+   * @deprecated As of 2.0.0-beta.1. Please use {@link app.notifyAppLoaded(): void} instead.
    * Notifies the frame that app has loaded and to hide the loading indicator if one is shown.
    */
   export function notifyAppLoaded(): void {
     app.notifyAppLoaded();
   }
 
   /**
-   * @deprecated with TeamsJS v2 upgrades
+   * @deprecated As of 2.0.0-beta.1. Please use {@link app.notifySuccess(): void} instead.
```

Comment:
I'm not sure what the right way is, but this doesn't seem to produce the results you'd expect:

![image](https://user-images.githubusercontent.com/6841973/142134634-7d791c24-5fe5-4011-aefc-1f5c3543f802.png)

Reply:
That probably applies to a lot of the comments you have here.

Reply:
I think what we actually want is simples:

```
@deprecated As of 2.0.0-beta.1. Please use {@link app.notifySuccess} instead.
```

That shows the correct comment when somebody tries to use the deprecated function.

Reply:
So yeah, I believe you'll need to update these not to use the full signature. After that you should validate in VSCode that the warnings show up with the right documentation.

Reply:
> I think what we actually want is simpler:
> 
> ```
> @deprecated As of 2.0.0-beta.1. Please use {@link app.notifySuccess} instead.
> ```
> 
> That shows the correct comment when somebody tries to use the deprecated function.

For some APIs, for example: `viewImages(uriList: ImageUri[]): Promise<void>`,  the way to make this API call is the same as the old version, but it cuts one param and returns a Promise now. I thought using the full signature would be more accurate and avoid confusion to developers in this situation. Do you think we should apply to other APIs for consistency?
Now I use:
`@deprecated As of 2.0.0-beta.1. Please use {@link media.viewImages media.viewImages(uriList: ImageUri[]): Promise<void>} instead.`
![viewImage](https://user-images.githubusercontent.com/85649572/142140406-8d44649d-33bc-4095-a22f-f81a07c48a74.png)